### PR TITLE
Notes on TH

### PR DESCRIPTION
**Pull request by Ericson2314**

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -273,12 +273,19 @@ similarly to plugins.
 
 It should have dynamic access (i.e. not via CPP) to the target platform
 properties (word size, endianness, etc.).
+@Ericson2314 thinks this is laudible, but not yet possible in all cases.
+In particular, metaprograms need to be able quote syntax with identifers resolved against the next stage's imports.
+To do that dynamically, we would need to interleave much of work of the GHC frontend with metaprogrmaming evalation / configuration choices.
+At HIW 2019, @Ericson2314 and @mpickering sketched out what we hope is prior work that shoes it is possible to do this.
```
Comment (hsyl20) [inline index.rst]: Do you have a link to this work?

Comment (Ericson2314) [reply index.rst]: @mpickering should we publicly share the google slides?

Comment (hsyl20) [reply index.rst]: I have found https://www.youtube.com/watch?v=YupkE1vsZ4o

Comment (Ericson2314) [reply index.rst]: Oh nice, I didn't know that was up.

**Pull request closed without merging**